### PR TITLE
Add React Components To Master File

### PR DESCRIPTION
**Pull request by WormJim**
Some relevant React components were added to Develop branch, and are ready to be pulled into Master

### CODE REVIEW

File: client/container/EventDetail.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        console.log('TCL: fetchFromDB -> data', data);
+        cb(data);
+      })
+      .catch((err) => {
+        console.log('TCL: fetchFromDB -> err', err);
+      });
+  };
+
+  return (
+    <>
```
Comment (CodeWritingCow) [inline client/container/EventDetail.js:33]: Should the empty brackets `<>` be `<div>` instead?

**Review by CodeWritingCow — CHANGES_REQUESTED** [review]
Are empty brackets `<>` valid syntax in HTML or React? If not, maybe you should change it to `<div>`.

File: client/container/EventDetail.js (hunk 1 above)
Comment (WormJim) [reply client/container/EventDetail.js:33]: Ah, no.  That's valid `JSX.`  

the `<>` and `</>` is sugar for `<Fragment />`  or `<React.Fragment />`.

[Here is a reference for you ](https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html)

Any other thoughts?

Comment (CodeWritingCow) [reply client/container/EventDetail.js:33]: No, that was the only thing that caught my eyes.

**Review by CodeWritingCow — APPROVED** [review]

**Pull request merged**